### ISSUE TITLE
[vue-3] useLoading is missing from exports

### ISSUE THREAD

**Issue by arrownorth** [ankurk91/vue-loading-overlay#88]
**I'm submitting a ...**  (check one with "x")
```
[ ] Bug report => search github for a similar issue or PR before submitting
[X ] Feature request
[ ] Other, please describe

I am just looking for an example using the new <script setup> feature in Vue 3.x. I can't figure out how to make this work. I see the code for useLoading but it is returning undefined when I do:

import { useLoading } from 'vue-loading-overlay';
const { loading } = useLoading();

**Comment by ankurk91** [comment]
https://github.com/ankurk91/vue-loading-overlay/blob/82716aee5d09884d18b065a715977f2c9be74a56/src/index.js#L14

Already exporting this, but webpack strip out the unused exports, need to find a solution for this

**Comment by arrownorth** [comment]
Should have said, I am using Vue + Vite. Vite uses Rollup and not Webpack I think. If there is anything I can do to help test/tweak, just let me know! Will continue poking at this (I am new to Vite & Vue).

**Comment by ankurk91** [comment]
I meant my webpack, not yours

**Comment by reslear** [comment]
need migrate to rollup, but now can use temp solution https://github.com/ankurk91/vue-loading-overlay/pull/91

**Comment by ankurk91** [comment]
I don't have time to migrate to roll up.
Damn busy now a days

**Comment by ankurk91** [comment]
tagged 4.0.4, please check and let me know

**Comment by ankurk91** [comment]
https://github.com/webpack/webpack/issues/2933

**Comment by ankurk91** [comment]
I have just released a beta version with named exports

```
npm install vue-loading-overlay@5.0.0
```
https://unpkg.com/browse/vue-loading-overlay@5.0.0/

i will mark it stable after more testing.

**Comment by ankurk91** [comment]
released 5.0.1
